lyvely-angular-code-review/src/app/app.component.html

```diff
@@ -0,0 +1,108 @@
+<style>
+  :host {
+    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+    font-size: 14px;
+    color: #333;
+    box-sizing: border-box;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+    background-color: aqua;
+  }
+
+  .background-sky {
+    width: 800px;
+    height: 800px;
+    border-radius: 1em;
+    margin: auto;
+    position: relative;
+  }
+
+  .background-sky.night {
+    background: rgb(26, 49, 118);
+    background: linear-gradient(180deg, rgba(26, 49, 118, 1) 0%, rgba(89, 163, 245, 1) 100%);
+  }
+
+  .background-sky.day {
+    background: rgb(2, 155, 223);
+    background: linear-gradient(180deg, rgba(2, 155, 223, 1) 0%, rgba(164, 229, 249, 1) 100%);
+  }
+
+  .background-sky .circle {
+    margin: auto;
+  }
+
+  .background-sky .text {
+    position: absolute;
+    bottom: 25%;
+    font-size: 30px;
+    color: white;
+    width: 100%;
+    text-align: center;
+    font-style: italic;
+  }
+
+  .background-sky .text-switch {
+    position: absolute;
+    bottom: 20%;
+    font-size: 16px;
+    color: #EEEEEE;
+    width: 100%;
+    text-align: center;
+    font-style: italic;
+  }
+
+  .background-sky .text-switch a {
+    color: #EEEEEE;
+  }
+
+  .background-sky.day .dark {
```

Comment:
we can get make this code a little bit more concise by using ngIf. We can declare a variable in the component which can track the state (day or night example isNight), and then use it to conditionally render the appropriate sun or moon image.  So in the template below add the ngIf for both sun and moon images with the corresponding condition